src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/GUID.txt

```diff
@@ -30,4 +30,14 @@ Error({Kind:ErrorKind.InvalidArgument})
 Error({Kind:ErrorKind.InvalidArgument})
 
 >> GUID("thisisastringthatismuchlongerthanaguidshouldbeitisquitealongstringandwillfinderrorswithlongstringsinthisfunction")
-Error({Kind:ErrorKind.InvalidArgument})
\ No newline at end of file
+Error({Kind:ErrorKind.InvalidArgument})
+
+>> IsBlank(GUID())
+false
+
+>> IsError(GUID())
+false
+
+// Multiple invocations are unique
+>> GUID() <> GUID()
+true
```

Comment:
Another case:
Lower(GUID())   // ensures type coercion for guid/string 

Len(GUID) = 36